src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Extensions/PositionExtensions.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Razor.LanguageServer.RazorLS;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.Extensions.Logging;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor.Extensions
+{
+    internal static class PositionExtensions
```

Comment:
This class was moved into the extensions folder. I also added the `TryGetAbsoluteIndex` method, but this was taken from another pre-existing extensions class.

Reply:
I think I did the same or similar thing.. lets see who wins the conflict war :D